Gemfile

```diff
@@ -48,6 +48,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri windows]
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
 end
 
```

Comment:
![image](https://github.com/anshuman7negi/Blog-app/assets/93039359/b1c412c9-f2a8-4670-9370-2ed95990c3b1)

- [ ] It would be nice if you fix the link highlighted in the image above 👀 because right now it is broken 😨 Kindly fix it or remove it `(If you don't have any dedicated section for this link in you readme file)` to make your readme descriptive and professional 👍